core/controllers/editor_test.py

```diff
@@ -554,8 +554,8 @@ def test_exploration_download_handler_for_default_exploration(self):
 
         # Check download to JSON.
         exploration.update_objective('Test JSON download')
-        exp_services._save_exploration(  # pylint: disable=protected-access
-            owner_id, exploration, '', [])
+        exp_services.update_exploration(
+            owner_id, exploration.id, [], '')
```

Comment:
This won't change the `objective` of the exploration, right? (You have to add a change command in the change list to update the objective of the exploration.)